src/packages/files/files.ts

```diff
@@ -89,6 +89,58 @@ export default (app: Express) => {
     }
   });
 
+  /**
+   * @swagger
+   * /file/{id}:
+   *   get:
+   *     parameters:
+   *       - description: ID of the file
+   *         in: path
+   *         name: id
+   *         required: true
+   *         schema:
+   *           type: number
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Fetches the file by id
+   *     summary: Get the file with the associated ID.
+   *     tags:
+   *       - Files
+   */
+  app.get('/file/:id', filesLimiter, auth, async (req, res) => {
+    const { params } = req;
+    const { id } = params;
+
+    try {
+      const { rows, rowCount } = await db.query(
+        `
+        select *
+        from merlin.uploaded_file
+        where id = $1;
+      `,
+        [id],
+      );
+
+      if (rowCount && rowCount > 0) {
+        const [row] = rows;
+        const { name } = row;
+
+        const filePath = path.join(FILE_PATH, name);
```

Comment:
In terms of security, do we need to worry about a request where the user goes outside the bounds of the specified file directory? This might typically be flagged as a vulnerability by IT.

Reply:
Why not use the `uploaded_files` table instead of having the user specify the path? That table has where the file lives and avoids this security concern.

Reply:
This does use the `id`, it never uses a user provided path or filename.